src/literal.js

```diff
@@ -0,0 +1,10 @@
+const run = require('./string/run')
+const text = require('./text')
+
+function literal (node, context) {
+  const raw = text(node.children || [])
+  if (raw) return run(raw)
+  else return null
```

Comment:
I think it will never be executed since, because [] => truthy

Reply:
Is that about the fallback to `[]`? I think this one is OK. That `text()` routine extracts text from a list of nodes. If it gets an empty list it gives you an empty result.

Reply:
what the `empty result` means? Is it `[]` or `null` or `''`?

Reply:
Looks like it's `''`. https://github.com/bookmoons/jmeter-to-k6/blob/dabbe3a6fd2f96db1c931043cdf41cfa9cc2fd53/src/text.js#L7